src/core/SkPixelRef.cpp

```diff
@@ -24,6 +24,9 @@ uint32_t SkNextID::ImageID() {
     do {
         id = nextID.fetch_add(2, std::memory_order_relaxed);
     } while (id == 0);
+
+    id = SkRecordReplayValue("SkNextID::ImageID", id);
```

Comment:
This diverges almost all the time, no matter where, so we assume this to be an acceptable divergence and just replay the value instead.